cirro/models/process/process.py

```diff
@@ -0,0 +1,136 @@
+import os
+from os import path
+from functools import cached_property
+import json
+from typing import Any, Iterable, Optional
+import logging
+
+from referencing import Resource, Registry
```

Comment:
This should be listed in the dependencies if it's used directly

Reply:
`referenceing` is a subdependency of `jsonschema`